DuckDuckGo/Preferences/Model/VPNPreferencesModel.swift

```diff
@@ -118,7 +123,6 @@ final class VPNPreferencesModel: ObservableObject {
         showInMenuBar = settings.showInMenuBar
         showInBrowserToolbar = pinningManager.isPinned(.networkProtection)
         showUninstallVPN = defaults.networkProtectionOnboardingStatus != .default
-        showExcludedSites = true
```

Comment:
This is old stuff that was really being ignored.  I'm using this property oportunistically for what I'm doing here, but this initialization is no longer needed.